templates/careers/commercial-operations.html

```diff
@@ -43,10 +43,11 @@ <h2 class="p-text--x-small-capitalised u-align-text--x-small-to-default">Working
   <div class="p-strip is-shallow u-no-padding--top">
     <div class="row">
       <div class="col-6">
-        <h3 class="p-heading--2">Operations at Canonical</h3>
+        <h3 class="p-heading--2">Commercial Operations at Canonical</h3>
       </div>
       <div class="col-6">
-        <p>Getting it done right is the best feeling in the world. No shortcuts, no loose ends, no waste. It's only possible to operate with mastery if you understand the domain deeply, you prioritise re-use and iteration, and you bring a sharp mind to the problem. A well-run operation invests in improving its data and its tools to elevate its self-awareness and automate its processes. Fighting fires is the exception, not the norm, and those moments are an opportunity to learn and improve.</p>
+        <p>At Canonical, we don't just run a great business, we run the infrastructure and applications that power many great businesses. From IT to commercial operations and logistics, our Commercial Operations team ensures that everything runs smoothly. Getting it done right is the best feeling in the world. No shortcuts, no loose ends, no waste.</p>
+        <p>We believe it's only possible to operate with mastery if you understand your domain deeply, prioritise re-use and iteration, and bring a sharp mind to every problem. That’s why we invest in improving our data and tools to elevate our self-awareness and automate processes. </p>
```

Comment:
```suggestion
        <p>We believe it's only possible to operate with mastery if you understand your domain deeply, prioritise re-use and iteration, and bring a sharp mind to every problem. That’s why we invest in improving our data and tools to elevate our self-awareness and automate processes. </p>
```
This text has been broken out into its own section, titled 'Projects you'll work on' in the copydoc